06-loops-and-functions.md: update keypoints (#403)
datacarpentry/python-ecology-lesson: _episodes/06-loops-and-functions.md

    - "Build reusable code in Python."
    - "Write functions using conditional statements (if, then, else)."
keypoints:
    - "Loops help automate repetitive tasks over sets of items."

>     - "Loops can be combined with functions we have learned so far to make data analysis and processing more efficient."

Reviewer: ....Um, not sure if we need this one, but:

Loops and functions are fundamental to data analysis.

(???)

Author: I think that belabors the point slightly, no?

Reviewer: Slightly? :)

I'm not sure what `make data analysis [snip] efficient` means...

Author: What I'm really trying to get at is that loops are versatile and can be combined with all the junk they learned before this lesson to do more science in less time.

Reviewer: > What I'm really trying to get at is that loops ... [snip] to do more science in less time

I think keypoints have to be about concrete stuff that is covered in the lesson, not a summary. I suggest that we either remove this keypoint or rephrase it using the text that you used above (without the "junk" part :) ).  BUT, given the time constraints imposed on us, feel free to ignore this comment and merge it as is.

Author: ```suggestion
    - "Loops can be combined with functions we have learned so far to process data more efficiently than can be accomplished by hand."
   ```
```

Author: Maybe like so?

Reviewer: I'd get rid of `can be` and `we have learned so far` 

Loops combined with functions provide a way to...

Author: brevity is the soul of wit. 

Not sure what that makes me ;)

>     - "Conditional statements allow us to sort data and perform different operations on different subsets of the data."

Reviewer: Conditional statements enable execution of different operations on different data.

Author: ```suggestion
    - "Conditional statements enable execution of different operations on different data."
```

>     - "Functions can enable code reuse."

Reviewer: Functions  enable code reuse

Author: ```suggestion
    - "Functions enable code reuse."
```
